src/ralph/lib/transitions/models.py

```diff
@@ -238,10 +241,9 @@ def run_field_transition(
     """
     Execute all actions assigned to the selected transition.
     """
-    if not isinstance(instances, Iterable):
+    if not isinstance(instances, list):
```

Comment:
This concept is not good.
Because they function inside run_field_transition we can not generate a new list because we lose a reference to it and redirect is will not work.

 I change it to run_field_transition received only a list, then the problem will disappear.